webui/react/src/components/ColumnPickerMenu.tsx

```diff
@@ -165,24 +188,44 @@ const ColumnPickerTab: React.FC<ColumnTabProps> = ({
   const rows = useCallback(
     ({ index, style }: { index: number; style: React.CSSProperties }) => {
       const col = filteredColumns[index];
+      const colType = col.type.replace('COLUMN_TYPE_', '');
+      const getColDisplayName = (col: ProjectColumn) => {
+        return (
+          <>
+            {col.displayName || col.column} <Badge text={colType} />
+          </>
+        );
+      };
```

Comment:
please make sure the "known unspecified" columns (archived, parent archived, part of search) are badged as boolean to prevent confusion